packages/x-tailwindcss/src/x-tailwind-plugin/components/button-group/default.ts

```diff
@@ -0,0 +1,31 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the default styles for component `button-group`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function buttonGroupDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    display: 'inline-flex',
+    flexFlow: 'row nowrap',
```

Comment:
This is the default behavior, do we have to specify it?

Reply:
Yep, we can remove it.